test/utils/test-environment.js

```diff
@@ -54,6 +61,14 @@ class MongoEnvironment extends NodeEnvironment {
       headers: headersWithValidTokenForLola
     };
 
+    const headersWithNoEmailTokenForMarv = {
```

Comment:
remove Marv -see comments about not using a name unless it has a function / communicates an intention.